content/docs/studio/how-it-works.md

```diff
@@ -1,48 +1,58 @@
 # How Iterative Studio Works
 
-[Iterative Studio](https://studio.iterative.ai/) works with the data, metrics,
-hyperparameters and model metadata that you add to your ML project Git
-repositories. It works very closely with your Git ecosystem.
+[Iterative Studio](https://studio.iterative.ai/) works very closely with your
+Git ecosystem. It parses the data, metrics, hyperparameters and model metadata
+that you add to your ML project Git repositories, and renders them on a web UI.
 
-This video illustrates how Iterative Studio works closely with your Git
-ecosystem.
+Additionally, you can [track metrics from your experiments in real
+time][live-metrics-and-plots] by pushing the metrics directly to Iterative
+Studio using [DVCLive].
```

Comment:
This seems like a general description. Why does it now link specifically to the one specific feature but nothing else from the list in the first paragraph? I think we could keep it general here e.g. avoid the new paragraph, just add "...parses the data and metrics in real time, hyperparams..."

Reply:
Thanks